lld/test/ELF/aarch64-feature-gcs.s

```diff
@@ -49,10 +49,28 @@
 # REPORT-WARN: warning: func2.o: -z gcs-report: file does not have GNU_PROPERTY_AARCH64_FEATURE_1_GCS property
 # REPORT-ERROR: error: func3.o: -z gcs-report: file does not have GNU_PROPERTY_AARCH64_FEATURE_1_GCS property
 
+## gcs-report-dynamic should report any dynamic objects that does not have the gcs property. This also ensures the inhertance from gcs-report is working correctly.
+
+# RUN: ld.lld func1-gcs.o func3-gcs.o no-gcs.so force-gcs.so -z gcs-report=warning -z gcs=always 2>&1 | FileCheck --check-prefix=REPORT-WARN-DYNAMIC %s
```

Comment:
Can we add a test case with only inputs that have the marking `force-gcs.so` and `-zgcs-report=error` and `-zgcs-report-dynamic=error`? We expect no error.

Reply:
lld runs quickly, but 10+ invocations are too much for a relatively minor feature. Can the tests be simplified?